### PR TITLE
Milestone Message with updated anchor id and FxVersion targeting

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

**Comment by punamdahiya** [comment]
I am not able to run `make check` and `make export` on my local , gives jsonschema not found even though `jsonschema 4.17.3`  and `python3` is installed via brew install

Error seen on `make check`
 import jsonschema
ModuleNotFoundError: No module named 'jsonschema'



Error seen on `make export`

scripts/export-all.py
pyenv: python: command not found

Including @brennie for feedback on above errors thanks!

**Comment by brennie** [comment]
>  I am not able to run `make check` and `make export` on my local , gives jsonschema not found even though `jsonschema 4.17.3`  and `python3` is installed via brew install
> 
> Error seen on `make check`
>  import jsonschema
> ModuleNotFoundError: No module named 'jsonschema'
> 
> 
> 
> Error seen on `make export`
> 
> scripts/export-all.py
> pyenv: python: command not found

You mentioned python is installed by brew but this error comes from pyenv. You don't want both brew python and pyenv installed. If you're going to use pyenv then pyenv should be managing your python installations.

Additionally, the jsonschema brew package is deprecated (https://formulae.brew.sh/formula/jsonschema). Brew shouldn't be managing your python packages, you will want to use pip for that.

If you uninstall brew python and install python through pyenv, *then* install jsonschema et al with pip the problem should go away.

**Comment by punamdahiya** [comment]
> Additionally, the jsonschema brew package is deprecated (https://formulae.brew.sh/formula/jsonschema). Brew shouldn't be managing your python packages, you will want to use pip for that.
+1 uninstall jsonschema using brew and re-install using pip worked

> 
> If you uninstall brew python and install python through pyenv, _then_ install jsonschema et al with pip the problem should go away.

`make export` uses python versions that support versions with python command e.g. `3.8.16` where as python3 is needed by `make` script . Setting both python versions using ` pyenv global system 3.8.16  ` on my local helped run `make export` script

**Comment by punamdahiya** [comment]
Updated patch with `InflightAssetsMessageProvider.sys.mjs` after running `make export` that can be used to check in to m-c. @Mardak please review thanks!

File: outgoing/InflightAssetsMessageProvider.sys.mjs [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+            "starzplay.com",
+            "www.sling.com",
+            "sling.com",
+            "www.facebook.com",
+            "facebook.com"
+          ]
+        }
+      },
+      {
+        "id": "WNP_MOMENTS_12",
```
Comment (punamdahiya) [inline outgoing/InflightAssetsMessageProvider.sys.mjs:242]: I don't think these moments message are live any more, will open a separate pull request moving these moments messages to archive

**Review by Mardak — APPROVED** [review]
Matches the on train message landing in https://bugzilla.mozilla.org/show_bug.cgi?id=1859928 except for version checks. Just making sure we have somewhere documented that we're good with stopping the message for 87-114?

**Comment by punamdahiya** [comment]
> Matches the on train message landing in https://bugzilla.mozilla.org/show_bug.cgi?id=1859928 except for version checks. Just making sure we have somewhere documented that we're good with stopping the message for 87-114?

Thanks! Just updated bug https://bugzilla.mozilla.org/show_bug.cgi?id=1859928#c9 documenting this change.

**Pull request merged**